lib/client/kubesession.go

```diff
@@ -201,12 +197,47 @@ func (s *KubeSession) pipeInOut(stdout io.Writer, mode types.SessionParticipantM
 	go func() {
 		defer s.cancel()
 
-		handleNonPeerControls(mode, s.term, func() {
-			err := s.stream.ForceTerminate()
-			if err != nil {
-				fmt.Printf("\n\rError while sending force termination request: %v\n\r", err.Error())
+		if mode == types.SessionPeerMode {
+			// copy from the local input to the remote shell:
+			buf := make([]byte, 1024)
+
+			stdin := s.term.Stdin()
+			if s.term.IsAttached() {
+				stdin = escape.NewReader(stdin, s.term.Stderr(), func(err error) {
+					switch err {
+					case escape.ErrDisconnect:
+						fmt.Fprintf(s.term.Stderr(), "\r\n%v\r\n", err)
+					case escape.ErrTooMuchBufferedData:
+						fmt.Fprintf(s.term.Stderr(), "\r\nerror: %v\r\nremote peer may be unreachable, check your connectivity\r\n", trace.Wrap(err))
```

Comment:
Nit: I don't think trace.Wrap does anything here (and below as well).

Reply:
Plus, `s.term.Stderr()` is user-visible, shouldn't be sending stack traces to users.

Reply:
This is the current behaviour of tsh ssh, should we get rid of this in both places?